demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,220 @@
+```
+The parameter description is as follows.
+- **pretrained_model:** Incoming pre-trained model
+- **max_iters:** Total training rounds. If a pre-trained model is used, the training round required for the quantized model is much smaller than normal training.
+- **LeaningRate.base_lr:** Adjust `base_lr` according to the total` batch_size`, the size of the two is positively related, and can be adjusted in proportion simply.
+- **LearningRate.schedulers.PiecewiseDecay.milestones: ** Please adjust it according to the change of batch size.
+- **num_epochs:** Train more epochs, the accuracy will be higher theoretically
+
+Insert the quantization and dequantization OP stages in the Program. If the user needs to change the quantization strategy, he can change the `config.yaml` configuration. We currently recommend the following configuration to obtain the best accuracy.
+
+```
+config = {
+         'weight_quantize_type': 'channel_wise_abs_max',
+         'activation_quantize_type': 'moving_average_abs_max',
+         'quantize_op_types': ['depthwise_conv2d', 'mul', 'conv2d', 'elmentwise_add', 'pool2d']
+     }
+```
+If you want to understand the meaning of each parameter, please refer to [PaddleSlim quant_aware API](https://paddlepaddle.github.io/PaddleSlim/api/quantization_api/#quant_aware)
+
+**Note:**
+- Because the quantification needs to modify the Program, some training strategies that will modify the Program need to be closed. `` sync_batch_norm '' and quantized Doka training will be wrong when used at the same time, the reason is unknown, please make sure to close it in the code. (Closed in the sample code)
+```
+build_strategy.fuse_all_reduce_ops = False
+build_strategy.sync_batch_norm = False
+```
+- In `train_image_classification.py`, `paddleslim.quant.convert` is mainly used to change the order of quantization op and inverse quantization op in Program. In addition, `paddleslim.quant.convert ` will also change the operator parameters such as `conv2d`,` depthwise_conv2d`, and `mul` to the values within the range of quantized `int8_t`, but the data type is still `float32`. This is the qat float32 model we need, the default position is `./quantization_models/act_*/float `.
+
+## 3. Convert fp32 qat model to MKL-DNN INT8 model
+The model saved after training in the previous step is the float32 qat model. We also need to remove the quantization, inverse quantization op, fuse some ops, and fully converted into INT8 model. Run the following script
+
+```
+python ./save_qat_model.py --qat_model_path=$PATH_TO_float32_qat_dir --int8_model_save_path=$PATH_TO_SAVE_INT8_MODEL --quantized_ops="conv2d,pool2d"
+```
+
+## 4. Inference test
+
+### 4.1 Data preprocessing
+To run the inference test, the data needs to be converted to binary first. Run the script without any pramaters to transform the complete ILSVRC2012 val data set to bianary file. Use `local` parameter to transform your own data.
```

Comment:
Maybe: "... Run the following script to transform the whole Imagenet validation dataset (ILSVRC2012) to a binary file. Add `--local` parameter to transform your own dataset instead."